portal/src/ScreenLayout.module.scss

```diff
@@ -0,0 +1,37 @@
+.root {
+  display: flex;
+  flex-direction: column;
+  height: 100vh;
+  width: 100vw;
+}
+
+.body {
+  /* flex: 1 0 0% is an idiom */
+  /* If it is flex: 1 0 0, we will have this bug https://github.com/philipwalton/flexbugs#flexbug-4 */
```

Comment:
Actually, is it fine to just drop IE support?

Reply:
Fluent UI supports IE 11. I think we can support IE 11 too.

Reply:
I read before that MS is [dropping IE support in their own product](https://news.ycombinator.com/item?id=24200864), and Fluent UI's support of IE11 & Edge Legacy seems [in legacy maintainence mode](https://github.com/microsoft/fluentui/wiki/Browser-Support). I don't think it's a good idea to continue supporting them anymore.

Reply:
Dropped support for IE 11 and we only support evergreen Edge now.